### ISSUE TITLE
Selenium Grid Extras broken for Chrome 74+

### ISSUE THREAD

**Issue by osmolyar** [groupon/Selenium-Grid-Extras#426]

**Comment by selfamit** [comment]
it should work, i am using it with chrome 75.0 and it works for me. I just made sure grid extras does not decide the chrome driver version for me.

For e.g. it will ask this question for driver version, it will automatically download it for you if you give the below version info, just check the download page, you will get the version number for each chrome releases. https://chromedriver.storage.googleapis.com/index.html?path=75.0.3770.90/

What version of Chrome Driver should we use?
Default Value: 75.0
75.0.3770.90
'75.0.3770.90' was set as your value

If its chrome 76, you if you give this version : 76.0.3809.25 , it will auto download the driver.
This should work.

**Comment by osmolyar** [comment]
Thanks for the tip!  Does Firefox work for you with grid extras also?

**Comment by osmolyar** [comment]
Hi @selfamit , thank you for the tip - but I'm still getting the same error running with Chrome Driver 75.0.3770.90/Chrome 75.0:

ERROR: [object Object]
10:38:42 chrome
10:38:42 Error

What OS/platform is your worker node on?  Also does it happen to work for you with Firefox?

**Comment by osmolyar** [comment]
@selfamit , thanks for your replies. One more question, what version of Webdriver are you running?  we have
    "webdriver": {
      "version": "3.141.59"
    },

**Comment by byamagata** [comment]
Is this problem still open? I am having an issue where it will not download anything since 75 as well.

**Comment by osmolyar** [comment]
@byamagata yes, the issue still exists.  Selenium-grid-extras won't download chromedriver versions past 2.46 (corresponding to browser version 73).  Driver versions after that follow a different format corresponding to the browser version.  Drivers can be downloaded manually, but so far all browser/driver/version/bitness combinations past 73(64-bit)/2.46(32-bit) result in the error  "invalid argument: 'text' must be a string" when setting text fields.
